File: Programming Basics/JavaScript/While loops/Labs/max-number.js
function solve(input) {
    let max = Number.NEGATIVE_INFINITY;

    let index = 0;
    while (input[index] != "Stop") {
        const currentNumber = Number(input[index]);
        if (currentNumber > max) max = currentNumber;

        index++;
    }

    console.log(max);
}

File: Programming Basics/C++/While loops/Labs/MaxNumber/MaxNumber.cpp
#include <iostream>
#include <limits.h>
#include <string>
using namespace std;

int main()
{
    int max = INT_MIN;

    string input;
    while (cin >> input && input != "Stop")
    {
        int currentNumber = stoi(input);
        if (currentNumber > max) max = currentNumber;
    }

    cout << max << endl;
}
